### DISCUSSION TITLE
Fix in dark note theme

### DISCUSSION THREAD

**Discussion by ArslanYM** [monkeytypegame/monkeytype#4234] [Ideas] [answered]
In the dark note theme(which is great), The left margin line as show in the below screenshot should align itself to the left of the text( as it is in an actual notebook). I feel like  it would look great! 
Any ideas?

![this](https://user-images.githubusercontent.com/104521101/235285054-23e68bd9-299d-45e8-95ac-d94fe5d6c5d3.png)

**Comment by Miodec** [comment, marked as answer]
Not really possible. There is no connection between the lines and the position of the words. The lines are basically just an image. Thats why they also dont line up vertically.

**Reply by ArslanYM** [reply]
Yeah but that means we can change the image somehow to make it aligned like a margin to the text

**Reply by tajhans** [reply]
@ArslanYM it can be a little to align the image because the image doesn't appear the same on all devices because not all devices have the same screen size.
